producer/kinesis.go

```diff
@@ -123,6 +134,8 @@ func (prod *Kinesis) Configure(conf core.PluginConfig) error {
 
 	prod.streamMap = conf.GetStreamMap("StreamMapping", "default")
 	prod.batch = core.NewMessageBatch(conf.GetInt("BatchMaxMessages", 500))
+	prod.recordMaxMessages = conf.GetInt("RecordMaxMessages", 1)
```

Comment:
I'll go with error.

On Thursday, May 19, 2016, Arne Claus <notifications@github.com> wrote:

> In producer/kinesis.go
> <https://github.com/trivago/gollum/pull/22#discussion_r63846764>:
>
> > @@ -123,6 +134,8 @@ func (prod *Kinesis) Configure(conf core.PluginConfig) error {
> >
> >  	prod.streamMap = conf.GetStreamMap("StreamMapping", "default")
> >  	prod.batch = core.NewMessageBatch(conf.GetInt("BatchMaxMessages", 500))
> > +	prod.recordMaxMessages = conf.GetInt("RecordMaxMessages", 1)
>
> Give an error if RecordMaxMessages is == 0 or force set it to
> shared.MaxI(1,conf.GetInt("RecordMaxMessages", 1)).
> Otherwise there could be a divide by 0 later.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/trivago/gollum/pull/22/files/a99564704a7b4be8b402a6bded94cdc3ee11ac3f#r63846764>
>